### PR TITLE
Update polkadot-v0.9.5 deps

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #519

### CODE REVIEW

**Review by shaunxw — COMMENTED** [review]
v0.9.5 is not released yet

**Comment by xlc** [comment]
v0.9.5 branch is created and we want to start using it ASAP

**Review by xlc — APPROVED** [review]

**Comment by shaunxw** [comment]
> v0.9.5 branch is created and we want to start using it ASAP

Okay, though it would be painful, as the deps branch update will be expected to break our compiling/CI from time to time and block our work.

**Comment by xlc** [comment]
it is not master, so hopefully won't be too bad

File: Cargo.dev.toml [hunk 1]
```diff
@@ -30,72 +30,72 @@ resolver = "2"
...
-sp-state-machine = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-std = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-storage = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-tasks = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-timestamp = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-transaction-pool = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-trie = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-version = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
-sp-wasm-interface = { git = "https://github.com/paritytech//substrate", rev = "1d04678e20555e623c974ee1127bc8a45abcf3d6" }
+frame-benchmarking = { git = "https://github.com/paritytech//substrate", rev = "c666a251691300c1651075a3b59ba1cf59c5a664" }
```
Comment (xlc) [inline Cargo.dev.toml:33]: Need to bump Substrate to `3bb42b9aaba65e956aea7aaef2c2fb3468fa3aaf`

**Review by xlc — CHANGES_REQUESTED** [review]

**Review by xlc — APPROVED** [review]

**Pull request merged**